sorc/chgres_cube.fd/search_util.f90

```diff
@@ -1,17 +1,19 @@
- module search_util
+!> @file
+!! @brief Replace undefined surface values.
+!!
+!! @author gayno NCEP/EMC
+!!
+!! Abstract: Replace undefined values with a valid value.  This can
+!!   happen for an isolated lake or island that is unresolved by
+!!   the input grid.
+!!
+!! Public Subroutines:
+!! -------------------
```

Comment:
Take out all instances of "-----------------------------------" or anything else that was put in as a text separator in the comments. The only acceptable separator is a single blank line.

Reply:
Done.